cdap-explore/src/main/java/co/cask/cdap/explore/executor/AbstractQueryExecutorHttpHandler.java

```diff
@@ -101,7 +101,7 @@ public int compare(QueryInfo first, QueryInfo second) {
     }
   }
 
-  protected String getCSVHeaders(List<ColumnDesc> schema)
+  protected static String getCSVHeaders(List<ColumnDesc> schema)
```

Comment:
Why change to static (not a big fan of static stuff). If you don't want sub-class to override, use `final`.